deployment/helm/tensorwave70b/scalarlm/values.yaml

```diff
@@ -1,14 +1,20 @@
 image:
-  repository: tensorwave/scalarlm-amd
-  tag: v0.82
+  repository: sudnya/scalarlm-amd
+  tag: v0.90
   pullPolicy: Always
 
 service:
   type: ClusterIP
-  api_port: 8000
-  vllm_port: 8001
+  api_port: 8200
```

Comment:
I prefer to keep these 82xx ports since it is currently deployed in the test namespace. Will update once deployed on llama70b namespace